calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/views/initialization_view.py

```diff
@@ -75,6 +85,49 @@ def __init__(self, calibrator: "CameraIntrinsicsCalibratorUI", cfg):  # noqa F82
         source_layout.addWidget(self.data_source_combobox)
         self.source_group.setLayout(source_layout)
 
+        self.params_combobox = QComboBox()
+        self.params_combobox.addItem("General", 0)
+        self.params_combobox.addItem("C1", 1)
+        self.params_combobox.addItem("C2", 2)
+        self.params_combobox.addItem("Ceres Calib", 3)
+        self.params_combobox.addItem("Load File", 4)
+
+        def on_params_combo_box_changed(index):
+            if self.params_combobox.currentText() == "Load File":
+                file_name, _ = QFileDialog.getOpenFileName(
+                    self, "Open File", "", "All Files (*.*);;Text Files (*.yaml)"
+                )
+                if file_name:
+                    print(f"Selected file: {file_name}")
```

Comment:
delete or use logging

Reply:
thanks, i will fix it

Reply:
f2ad5cce4cffc72284d0b575020c40851475872e

Reply:
This is not the commit that addressed the required changes